woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -1146,6 +1137,25 @@ def test_dataframe_methods_on_accessor_other_returns(sample_df):
         pd.testing.assert_index_equal(schema_df.ww.keys(), schema_df.keys())
 
 
+def test_dataframe_methods_on_accessor_to_pandas(sample_df):
```

Comment:
Nice test to add - should we also confirm that the dtypes for all the columns match the expected `primary_dtype` value after converting to pandas?

Reply:
Ah, interesting. Makes sense.

If this becomes an issue in the future, I think we could probably write a custom `to_pandas` method on the KoalasTableAccesor class to handle this, making sure the dtypes are converted properly.

Reply:
going to leave the test as is for now without the Categorical column for simplicity